tests/third_party/cupy/linalg_tests/test_decomposition.py

```diff
@@ -0,0 +1,248 @@
+import unittest
+
+import numpy
+import pytest
+
+import dpnp as cupy
+from tests.helper import has_support_aspect64
+from tests.third_party.cupy import testing
+from tests.third_party.cupy.testing import condition
```

Comment:
```suggestion
from tests.third_party.cupy.testing import _condition
```
and rename `tests/third_party/cupy/testing/condition.py` to `tests/third_party/cupy/testing/_condition.py`